### PR TITLE
Fix colorscheme

### PR DESCRIPTION
**Pull request by sesam**
Adjusts color scheme to match website. Please see commit message for more info.

### CODE REVIEW

File: sledilnik_mobile_app/lib/ui/widgets/info_box.dart [hunk 1]
```diff
@@ -33,7 +32,7 @@ class InfoBox extends StatelessWidget {
     final trendColor = colorScheme.getTrendColor(trendType);
     return Container(
         padding: const EdgeInsets.symmetric(horizontal: 8.0, vertical: 8.0),
-        decoration: BoxDecoration(color: Colors.yellow[50]),
+        decoration: BoxDecoration(color: Color(0xfffffbe8)),
```
Comment (MihaMarkic) [inline sledilnik_mobile_app/lib/ui/widgets/info_box.dart]: Can you move this color to CustomColorScheme and assign a name to it?

Comment (sesam) [reply sledilnik_mobile_app/lib/ui/widgets/info_box.dart]: Taken care of. I expect we'll need more changes later on, as the app evolves. For now I named it primaryYellow.

Comment (MihaMarkic) [reply sledilnik_mobile_app/lib/ui/widgets/info_box.dart]: Can you name this guy as well? `decoration: BoxDecoration(color: Color(0xfffffbe8)),`

**Review by MihaMarkic — CHANGES_REQUESTED** [review]
Name this guy as well? `decoration: BoxDecoration(color: Color(0xfffffbe8)),`

**Pull request merged**